test/e2e/runner/load.go

```diff
@@ -105,13 +105,27 @@ func loadGenerate(ctx context.Context, chTx chan<- types.Tx, size int64) {
 		case chTx <- tx:
 			// sleep for a bit before sending the
 			// next transaction.
-			waitTime := (50 * time.Millisecond) + time.Duration(rand.Int63n(int64(time.Second))) // nolint: gosec
-			timer.Reset(waitTime)
+			timer.Reset(loadGenerateWaitTime(size))
 		}
 
 	}
 }
 
+func loadGenerateWaitTime(size int64) time.Duration {
+	const (
+		min = int64(100 * time.Millisecond)
+		max = int64(time.Second)
+	)
+
+	var (
+		baseJitter = rand.Int63n(max-min+1) + min // nolint: gosec
```

Comment:
This is fascinating. It implies the problem was basically a thundering herd?

Reply:
pretty sure, yes.

Reply:
Were the failures caused by the legacy p2p stack, the new one or both. I'm just wondering if we want to document this as an issue so we're aware of it and can make sure we address it at a later point in time